index.android.js

```diff
@@ -22,6 +22,8 @@ import { fetchRegionFromStorage } from './src/actions/region';
 
 store.dispatch(fetchRegionFromStorage());
 
+var styles = require('./src/styles');
```

Comment:
Also, you can import styles using es6 import in all places where styles are imported.